doc/source/user_guide/transforming_image_data.rst

```diff
@@ -81,12 +81,24 @@ from RGB to grayscale::
 Converting a grayscale image to RGB with :func:`gray2rgb` simply
 duplicates the gray values over the three color channels.
 
+Image inversion
+~~~~~~~~~~~~~~~
+
+An inverted image is also called complementary image. For binary images, True values
+become False and conversely. For grayscale images, pixel values are replaced by the
+difference of the maximum value of the data type and the actual value. For RGB
+images, the same operation is done for each channel. This operation can be achieved
+with the negation operator ~ on any numpy array::
+
+    >>> img = data.camera()
+    >>> inverted_img = ~img
```

Comment:
Yes, and I would say that this is *not* the desired operation anyway. The inverted image should be the *dtype* maximum, minus the image.